src/winforms/toga_winforms/keys.py

```diff
@@ -18,33 +19,16 @@ def toga_to_winforms_key(key):
     Key.MOD_1: WinForms.Keys.Control,
     Key.MOD_2: WinForms.Keys.Alt,
 }
+WINFORMS_MODIFIERS_MAP.update({
+    getattr(Key, modifier.upper()): getattr(WinForms.Keys, modifier.title())
+    for modifier in ["shift", "up", "down", "left", "right", "home"]
+})
+
 WINFORMS_KEYS_MAP = {
     Key.PLUS.value: WinForms.Keys.Oemplus,
     Key.MINUS.value: WinForms.Keys.OemMinus,
```

Comment:
In testing this PR, I noticed that the plus/minus key shortcuts render in an unexpected way:

<img width="310" alt="Screen Shot 2020-09-28 at 9 39 35 am" src="https://user-images.githubusercontent.com/37345/94382230-8d162080-016e-11eb-9bc8-351d82c0a128.png">

Reply:
In Windows, there is a difference between +/- keys in the notepad and +/ keys in the number row, as described [here](https://stackoverflow.com/a/8238670/6624798). I think most users would use the later in favor of the first. maybe in a future commit, I'll make sure both are supported.

Reply:
Well *that* sucks...